controllers/index.go

```diff
@@ -37,6 +37,7 @@ func (ic *IndexController) Get(w http.ResponseWriter, r *http.Request) {
 		CurrentAndNext *myradio.CurrentAndNext
 		Banners        []myradio.Banner
 		Teams          []myradio.Team
+		MsgBoxPrompt   bool
```

Comment:
Seems to be used for an error message - variable poorly named? (should probably be explicitly initialised to false as well)